nautobot/dcim/tests/test_filters.py

```diff
@@ -2647,6 +2661,21 @@ def test_search(self):
         params = {"q": value}
         self.assertEqual(self.filterset(params, self.queryset).qs.values_list("pk", flat=True)[0], value)
 
+    def test_device_redundancy_group(self):
+        device_redundancy_groups = DeviceRedundancyGroup.objects.all()[:2]
+        with self.subTest():
+            params = {"device_redundancy_group": [device_redundancy_groups[0].pk]}
+            self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        with self.subTest():
+            params = {"device_redundancy_group": [device_redundancy_groups[0].name]}
+            self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
+        with self.subTest():
+            params = {
+                "device_redundancy_group": [device_redundancy_groups[0].pk],
+                "device_redundancy_group_priority": [1],
+            }
+            self.assertEqual(self.filterset(params, self.queryset).qs.count(), 1)
+
```

Comment:
Are we using `device_redundancy_groups[1]` here at all? It also looks like we only have one `DeviceRedundacyGroup` instance created.